packages/create-app/src/initialize.ts

```diff
@@ -68,24 +174,55 @@ export class Command extends clipanion.Command {
      * The actual {@link clipanion.Command} logic.
      */
     async execute(): Promise<void> {
-        const logDebug = makeLogger(this.verbose);
-        const logInfo = makeLogger(true);
-        const directory = this.directory ?? this.example;
+        const directory = this.directory ?? this.example ?? defaultExample;
 
-        logDebug({
+        this.logDebug({
             options: {
                 commit: this.commit,
-                example: this.example,
                 directory,
+                example: this.example,
+                features: this.features,
+                install: this.install,
                 verbose: this.verbose
             }
         });
 
-        logInfo(`Creating ${directory} directory`);
+        // We don't want to deal with both `--examples` and `--features` at the same time.
+        if (this.example != null && this.features != null) {
+            return Promise.reject(
+                new Error(
+                    `Cannot use both \`--example\` and \`--features\` at the same time. Please choose one or the other`
+                )
+            );
+        }
```

Comment:
I really appreciate the attention to detail you have with these edge-cases. I would have probably just let them both be specified, but not really do anything with it. Your way is better!